ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -314,6 +314,27 @@
                     current-parameters: true
                   - name: 'ceph-ansible-prs-dev-centos-container-add_mdss'
                     current-parameters: true
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is master we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ "stable-" ]]; then
```

Comment:
Although all of this isn't a bugfix, I think we will exceptionally have to backport all of this in stable-4.0